src/elisp/treemacs-extensions.el

```diff
@@ -447,28 +449,33 @@ additional keys."
                          ,(format "The handle to update the the %s extension." name))
                        (defun ,ext-name ()
                          (treemacs-with-writable-buffer
-                          (-let [pr (make-treemacs-project
+                          (let ((pr (make-treemacs-project
                                      :name ,root-label
                                      :path ,root-key-form
-                                     :path-status 'extension)]
+                                     :path-status 'extension))
+                                (button-start (point)))
                             (treemacs--set-project-position ,root-key-form (point-marker))
                             (setq-local ,project-var-name pr)
-                            (insert (propertize "a"
+                            (insert (propertize "\n"
                                                 'button '(t)
                                                 'category 'default-button
+                                                'invisible t
+                                                'cursor-sensor-functions (list (lambda (_ previous-position change)
+                                                                                 (when (eq change 'entered)
+                                                                                   (if (< previous-position (point))
+                                                                                       (treemacs-next-line 1)
+                                                                                     (treemacs-previous-line 1))
```

Comment:
Here are unhandled corner cases - if there are zero buttons, this will end up in an infinite loop. Same goes for wrapping around (or maybe it will just end up on the hidden button in that case)